src/content/docs/accounts/accounts/automated-user-management/okta-scimsso-application-configuration.mdx

```diff
@@ -80,13 +80,36 @@ Assigning users is done using two different tabs in the app. We recommend having
+Once provisioned, your New Relic users start out as [full users](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#user-type). If your organization is on [New Relic One pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), these users are billable. 
+
+To convert users to free [basic users](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#user-type), you have two choices: 
+
+* Manage user type from New Relic using the [User management UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles#add-users), OR
+* Manage user type from Okta (described below). 
+
+To manage your users' user type from Okta: 
+
+1. Go to the [authentication domain UI](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#manage-user-type-scim) and click **Enable Manage user type with SCIM**. Note that when this is enabled, you can’t manage user type from the New Relic UI and can only manage it from Okta.
+2. Next, you'll configure Okta to be able to send a new attribute `nrUserType`. Steps: 
+	* Go to the **Profile editor**. In the **Attributes** section, click **Add attribute**. 
+	* Set your settings to match the screenshot below (except for the variable name, which is created automatically). The only two fields that must match exactly are **External name** and **External namespace**. The value for **External namespace** must be `urn:ietf:params:scim:schemas:extension:newrelic:2.0:User`
+	
+  ![Adding new attribute in Okta app](./images/new-relic-okta-app-add-user-type.png "Adding new attribute in Okta app")
+
+3. Next, you'll configure your Okta user profile to have this field. Steps: 
+	* In the **Profile editor**, go to **Users** and click the **User (default)** profile. 
+	* Add a new **New Relic user type** attribute to that profile ([see Okta user profile instructions](https://developer.okta.com/docs/concepts/user-profiles/#universal-directory-schemas)). How you set this will depend on your own setup and preferences for defining user type (e.g., using an enumerated list). Note that the expected values for user type are `Basic user` and `Full user`. Below is an example with information filled in. 
+  ![Set Okta user profile to have new attribute](./images/okta-side-define-users-type.png "Set Okta user profile to have new attribute")
```

Comment:
I find the text in this image pretty tough to read.

Reply:
If there's specific info that you want people to know, maybe pull it out into text? Maybe in a caption or something, I don't know.